skipruntime-ts/ts/examples/database.ts

```diff
@@ -100,32 +82,35 @@ async function update(
   event: TJSON,
   writers: Record<string, Writer<string>>,
 ): Promise<void> {
-  const writer = writers["users"];
-  const cmd = event as Command;
-  switch (cmd.command) {
-    case "set": {
-      for (const entry of cmd.payload) {
-        await db.run(
-          "INSERT OR REPLACE INTO data (id, object) VALUES ($id, $object)",
-          {
-            $id: entry.key,
-            $object: JSON.stringify(entry.value),
-          },
-        );
-        writer.set(entry.key, entry.value);
+  await new Promise((resolve) => {
```

Comment:
Do we really need this? I think we don't

Reply:
I don't know. On one hand, the code before this PR is just type-incorrect as the function returns `void` and should return `Promise<void>`. And, not that I trust it, but the linter complains. On the other hand, returning a promise that immediately resolves seems superfluous. Initially I just wanted to use the `return` operation of the `Promise` monad, but there doesn't seem to be one in JS. Would putting `await new Promise((resolve) => resolve(undefined))` at the end instead of putting the body of this code in the resolve continuation make you happier?

Reply:
The other changes in this commit are the ones that fix the failures and odd behavior of the test that I commented about earlier (cc @pikatchu), but FTR if I omit the `await new Promise` parts, I still cannot reproduce the test failures.

Reply:
My personal view is that I would rather go with one of the type-correct options, even if we cannot observe a runtime failure with the option that has less code. Also, in this case the linter was able to detect a real issue that the type-checker failed to notice, so I am inclined to leave it enabled, which also supports going with one of the type-correct options.

Reply:
I think we should change the return type of `update` in (`Simple`)`ServiceOutput` to `void | Promise<void>`. It is OK to `await` a non-`Promise`, it's a no-op.

Reply:
Like https://github.com/SkipLabs/skdb/pull/333?

Reply:
Do we still need the `await new Promise ...` in this commit?

Reply:
I will squash the commits that intro and elim `await new Promise` and get the ones in sum and sheet too.